.appveyor.yml

```diff
@@ -28,7 +28,7 @@ install:
   - "pip install -U tox"
 
 test_script:
-  - ps: "tox -- --cov mitmproxy --cov pathod -v"
+  - ps: "tox -- --verbose --cov-report=term"
```

Comment:
For my new feature to work, we need `--cov` which by default adds the term-report. However, just running tox, I don't want to display it. Therefore I disabled the report in tox explicitly. On Travis+Appveyor I override it, because I DO want to see it for CI reasons.